katas/content/index.json

```
@@ -14,5 +14,6 @@
   "oracles",
   "deutsch_algo",
   "deutsch_jozsa",
-  "qec_shor"
+  "qec_shor",
+  "distinguishing_states"
```

Remove before merging
```suggestion
  "qec_shor"
```